demo/protocolgen/src/main/scala/service.scala

```diff
@@ -0,0 +1,40 @@
+package freestyle
+package rpc
+package demo
+package protocolgen
+
+import freestyle.internal.EffectLike
+import freestyle.rpc.protocol._
+import io.grpc.stub.StreamObserver
+
+@option(name = "java_package", value = "com.example.foo", quote = true)
+@option(name = "java_multiple_files", value = "true", quote = false)
+@option(name = "java_outer_classname", value = "Ponycopter", quote = true)
+object protocols {
+
+  @message
+  case class MessageRequest(name: String, n: Int)
+
+  @message
+  case class MessageReply(name: String, n: Int)
+
+  @service
```

Comment:
is `@service` annotation also generating (automatically) the algebra? Or, should the user annotate the service with `@free` tag?